pages/profile/myRecipes.js

```diff
@@ -75,99 +71,33 @@ export default function MyRecipes({
         onClick={() => router.back()}
       />
       <Spacer />
-
-      <StyledH2>
-        <div>Kochbücher</div>
-        <StyledLink href="/profile/collections">alle anzeigen</StyledLink>
-      </StyledH2>
-      <Wrapper>
-        {user.collections.length
-          ? user.collections.map((col, index) => (
-              <CollectionCard key={index} collection={col} />
-            ))
-          : `Du hast noch keine Kochbücher angelegt.`}
-        <StyledCollection onClick={toggleAddCollection}>
-          <BookPlus width={40} height={40} />
-          <StyledParagraph />
-        </StyledCollection>
-        {isModalCollection && (
-          <NewCollection
-            user={user}
-            mutateUser={mutateUser}
-            setModal={toggleAddCollection}
-          />
-        )}
-      </Wrapper>
       <StyledH2>Meine Rezepte</StyledH2>
-      <StyledArticle>
-        <StyledUl>
-          {myRecipes.length
-            ? myRecipes?.map((recipe) => {
-                return (
-                  <MealCard
-                    key={recipe._id}
-                    recipe={recipe}
-                    isFavorite={getRecipeProperty(recipe._id, "isFavorite")}
-                    onToggleIsFavorite={toggleIsFavorite}
-                  ></MealCard>
-                );
-              })
-            : "Du hast noch keine eigenen Rezepte erstellt."}
-        </StyledUl>
-      </StyledArticle>
+
+      <StyledUl>
+        {myRecipes.length
+          ? myRecipes?.map((recipe) => {
```

Comment:
You already check whether myRecipes.length is truthy one line prior to this, so you won't need this optional chaining:

```suggestion
          ? myRecipes.map((recipe) => {
```